src/main/java/org/embulk/output/mailchimp/helper/MailChimpHelper.java

```diff
@@ -83,7 +83,7 @@ public String apply(@Nullable JsonNode input)
             }
```

Comment:
the method `MailchimpHeloper#extractMemberStatus` is test helper method, consider moves it to the test package

Reply:
Also please moves the `acontainsCaseInsensitive`

Reply:
updated